x-pack/test/functional/es_archives/endpoint/metadata/api_feature/data.json

```diff
@@ -10,6 +10,11 @@
         "version": "6.6.1",
         "name" : "Elastic Endpoint"
       },
+      "elastic": {
```

Comment:
Question: did you update this es_archive manually? or did you use some sort of a tool that just add "stuff" to it?

Reply:
> Question: did you update this es_archive manually? or did you use some sort of a tool that just add "stuff" to it?

Yes we have to make incremental changes to the data because the changes are relative, e.g. in this case all events for a particular endpoint should have the same elastic.agent.id. And we have to be backward compatible with other tests too.

Reply:
So I assume you updated the file manually?

Reply:
Yes, I updated manually.